src/components/home/QuoteBox.tsx

```diff
@@ -1,7 +1,7 @@
 const QuoteBox = ({ quote, name }: { quote: string; name: string }) => {
   return (
-    <div className="flex justify-center">
-      <div className="font-lora relative w-3/5">
+    <div className="flex justify-center px-4 md:px-0">
```

Comment:
```suggestion
    <div className="flex justify-center">
```
using px does indeed make the quote box smaller, but instead we can just directly control the width of the quote box. so, decreasing the width and increasing the padding becomes redundant. so, we can just remove the padding from here